dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/TracesFeature.kt

```diff
@@ -52,8 +52,9 @@ internal object TracesFeature {
         clientToken = config.clientToken
         endpointUrl = config.endpointUrl
         serviceName = config.serviceName
+        val envSuffix = if (config.envName.isEmpty()) "" else ", \"env\": \"${config.envName}\""
```

Comment:
Shouldn't we escape `config.envName` value? We don't validate it anywhere, so the user can potentially spoil all traces by providing JSON-breaking character like `"`.

Reply:
We could, but then again because the env is a notion across Datadog, users are used to it being a simple value.